CONTRIBUTING.md

```diff
@@ -78,23 +60,12 @@ feature requests.
 
 ## Contributing a feature
 
-If you would like to propose a new feature for the project then it would be
-best to first discuss your idea with the community to gauge their level of
-interest. You can use any of the communication channels to have this
-discussion, but ideally a new GitHub issue should be opened so that the
-history of the discussions can be saved within the repository itself.
-The issue should include information about the requirements and
-use cases that it is trying to address.
-
-If you would like to also work on the implementation of the feature then
-it should include a discussion of the proposed design and technical details
-of how it will be implemented.
-
-Before checking in a new feature, you are expected to contribute full
-documentation for this feature within the main Istio repo (https://github.com/istio/istio).
-
-Once the idea has be discussed and there is a general agreement on the
-technical direction, a PR can then be submitted.
+In order to contribute a feature to Istio you'll need to go through the following steps:
+- Discuss your idea with the appropriate [working groups](wgs.md).
+- Once there is general agreement that the feature is useful, create a GitHub issue to track the discussion. The issue should include information about the requirements and use cases that it is trying to address.
+- Include a discussion of the proposed design and technical details of the implementation in the issue.
+- Contribute documentation of the feature to the main Istio repo (https://github.com/istio/istio).
```

Comment:
Oh I think you mean documentation after submitting PR?  will do.  Also will swap to istio.github.io.